tests/unit/Util/ZipFilesTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Tests\Util;
+
+use PHPUnit\Framework\TestCase;
+
+use Doctrine\Common\Collections\ArrayCollection;
+
+use App\Entity\File;
+use App\Entity\Fileset;
+use App\Util\ZipFiles;
+
+/**
+ * Unit tests for App\Util\ZipFiles.
+ */
+class ZipFilesTest extends TestCase

```

Comment:
This test case it not needed I think. It can be removed. Together with the test files.